screens/authentication/signUp/SignUpScreen.jsx

```diff
@@ -59,9 +59,10 @@ const SignUpScreen = ({navigation}) => {
     },
   });
 
```

Comment:
create initialValues variable outside from useFormik and wrap it inside useMemo and after it put that variable inside useFormik hook, the same do for function onSubmit and wrap inside useCallback


plus you have a valiadtionSchema instead of checking variables here if (data.email && data.password && data.name)
add validation fields in validationSchema